README.md

```diff
@@ -396,7 +396,8 @@ docker run -d -p 8080:80 -p 8021:21 -e "GALAXY_LOGGING=full" -v `pwd`/gx_logs:/h
 ### Basic setup for the filesystem  <a name="Basic-setup-for-the-filesystem" /> [[toc]](#toc)
 
 #### The easy way
-The easiest way is to create a /export mount point on the cluster and mount the container with /export:/export.
+The easiest way is to create a `/export` mount point on the cluster and mount the container with `/export:/export`.
+Also a symbolic link `galaxy-central` pointing to `/export/galaxy-central` needs to be created on the cluster.
```

Comment:
Do you mean `ln -s /export/galaxy-central /galaxy-central`? If so why is this needed?

Reply:
When you are changing case, or using some other default galaxy tool:

```bash
perl /galaxy-central/tools/filters/changeCase.pl /export/galaxy-central/database/files/000/dataset_1.dat "c1,c2" TAB up /export/galaxy-central/database/files/000/dataset_2.dat
```
So the /galaxy-central location is needed on the cluster.

Reply:
To be honest. I therefore think the "easy way" is actually not so easy. I prefer to change the configuration so you have full control on where the folders are. Also this means file permissions on the folder are much more clear.

Reply:
But galaxy-central is also in /export. We just need to convince Galaxy to use this as the path.

Reply:
That can be achieved by setting `GALAXY_CONFIG_TOOL_PATH=/export/galaxy-central/tools`

Reply:
Its a good idea the set this by default I think. I have done this in the dev branch, which will become 17.09 shortly.

Reply:
Changed the documentation accordingly.